px4_ros2_cpp/include/px4_ros2/navigation/experimental/local_position_measurement_interface.hpp

```diff
@@ -0,0 +1,145 @@
+/****************************************************************************
+ * Copyright (c) 2023 PX4 Development Team.
+ * SPDX-License-Identifier: BSD-3-Clause
+ ****************************************************************************/
+
+#pragma once
+
+#include <optional>
+#include <rclcpp/rclcpp.hpp>
+#include <Eigen/Eigen>
+
+#include <px4_msgs/msg/vehicle_odometry.hpp>
+#include <px4_ros2/navigation/experimental/navigation_interface_base.hpp>
+
+namespace px4_ros2
+{
+
+using AuxLocalPosition = px4_msgs::msg::VehicleOdometry;
+
+enum class PoseFrame
+{
+  Unknown,
+  NED,
+  FRD
+};
+
+enum class VelocityFrame
+{
+  Unknown,
+  LocalNED,
+  LocalFRD,
+  BodyFRD
+};
```

Comment:
@bresch Do you think we should also make the _pose_ frames `LocalNED` and `LocalFRD`? While there is no chance for confusion between local and body, it would be more consistent with the velocity frames.

Reply:
Yes, makes total sense